index.html

```diff
@@ -431,25 +434,25 @@ <h3 id="reading-current-states">
           Reading the current permission state
         </h3>
         <p>
-          A |descriptor|'s <dfn class="export" data-local-lt="state">permission state</dfn> for an
```

Comment:
You should keep a `<dfn>` here so that people can find the algorithm that produces the state for a given descriptor. It could be `<dfn for="permission descriptor">` if you don't think "permission state" is appropriately defined by an algorithm over descriptors.